src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: inline-block;
+}
+
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+}
```

Comment:
The 'main' class is too general. Consider renaming it to be more specific. Also, it would be good to move the styles for this class to a separate file, according to BEM methodology.